.github/workflows/cd-pipeline.yml

```diff
@@ -124,3 +124,75 @@ jobs:
           VIB_ENV_REGISTRY_URL: ${{ secrets.PROD_MARKETPLACE_REGISTRY_URL }}
           VIB_ENV_REGISTRY_USERNAME: ${{ steps.get-registry-credentials.outputs.marketplace_user }}
           VIB_ENV_REGISTRY_PASSWORD: ${{ steps.get-registry-credentials.outputs.marketplace_passwd }}
+      - name: Install s3cmd
+          run: sudo apt-get install -y s3cmd
+      - name: 'Getting 3rd party packages for OSSPI for ${{ steps.get-container-metadata.outputs.name }}: ${{ steps.get-container-metadata.outputs.tag }}'
+        if: ${{ steps.get-container-metadata.outputs.result == 'ok' }}
+        env:
+          # Path with docker resources
+          VIB_ENV_PATH: ${{ matrix.container }}
+        run: |
+          # If it's set from outside, can be changed
+          ARCH="${ARCH:-amd64}"
```

Comment:
I think we are fixing the architecture to `amd64` here, I don't see other option, is this intentional?

Reply:
This is thinking on a possible future extension